### ISSUE TITLE
Alert banner hide button aria-labels lack context - all 'Hide alert'

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_base#146]
[Accessibility audit ref 62](https://docs.google.com/spreadsheets/d/1t8rRIQe_1kogxPBZDKcsbf8YxtEfC9iGJvidkagNSXY/edit#gid=1779042480&range=D63)

The aria-label for the hide button on alert banners is set to 'Hide alert'. If there are multiple alert banners live at the same time, they all use the same aria-label, so there's no context for the user to know which alert the button will hide.

Suggest that the aria-label attribute is extended to include the alert title, eg: "Hide alert: Shakespeare is dead"

Related issue in localgov_alert_banner: https://github.com/localgovdrupal/localgov_alert_banner/issues/177

**Comment by waako** [comment]
Looks like this has been resolved in https://github.com/localgovdrupal/localgov_alert_banner/pull/178 (which closed https://github.com/localgovdrupal/localgov_alert_banner/issues/177).

This issue can be closed therefore ?
